packages/fluxcd-source-controller/0.33.0/package.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: fluxcd-source-controller.community.tanzu.vmware.com.0.33.0
+spec:
+  refName: fluxcd-source-controller.community.tanzu.vmware.com
+  version: 0.33.0
+  releaseNotes: "Adding placeholder secrets for authenticated registeries"
+  releasedAt: "2022-12-15T15:14:08Z"
+  valuesSchema:
+    openAPIv3:
+      properties:
+        namespace:
+          type: string
+          description: Deployment and service namespace
+          default: source-system
+        resources:
+          type: object
+          description: Pod resource limits
+          properties:
+             limits_cpu:
+               type: string
+               description: Set cpu usuage limit
+               default: 1000m
+             limits_memory:
+               type: string
+               description: Set memory usuage limit
+               default: 1Gi
+        service_port:
+          type: integer
+          description: Set service port
+          default: 80
+        proxy:
+          type: object 
+          description: Pod proxy configuration
+          properties:
+            no_proxy:
+              type: string
+              description: Set domains for which no proxying should be performed
+              default: ""
+            https_proxy:
+              type: string
+              description: Set secure proxy connection information
+              default: ""
+            http_proxy:
+              type: string
+              description: Set unsecure proxy connection information
+              default: ""
```

Comment:
you need to add all the three params here as well if we are following user to override these values.
#@ "--default-service-account=" + data.values.default_service_account
Please source controller proxy variables.
@narahari92 can user overwrite this values.

Reply:
Yes user should be able to overwrite these values

Reply:
source controller was complaining --default-service-account is a unknown flag in my testing, so I raised an issue in fluxcd upstream and got the answer that this flag just go removed, so I removed this flag in this PR as well

fluxcd issue link:
https://github.com/fluxcd/source-controller/issues/1002